registry/remote/credentials/store.go

```diff
@@ -193,8 +193,8 @@ func (ds *DynamicStore) getStore(serverAddress string) Store {
 	return fs
 }
 
-// getDockerConfigPath returns the path to the default docker config file.
-func getDockerConfigPath() (string, error) {
+// GetDockerConfigPath returns the path to the default docker config file.
+func GetDockerConfigPath() (string, error) {
```

Comment:
This method should not be exported as it is not in an internal package.

Reply:
Sorry I don't get it, it's not in an internal package. I need this to default path of docker config. I thought everything under `registry/remote/credentials` are exportable.